charts/fulcio/values.yaml

```diff
@@ -20,9 +20,14 @@ server:
     # crane digest gcr.io/projectsigstore/fulcio:v1.3.1
     # -- v1.3.1
     version: sha256:c920be2d367214562cda7d53d3af3529edf2dfd9326b24909ece450092b97b18
+  # env:
+  #   # Valid values: path to workload identity config or service account key JSON file
+  #   GOOGLE_APPLICATION_CREDENTIALS: /etc/fulcio-config/cloud_credentials
   args:
     port: 5555
     grpcPort: 5554
+    # valid values: GCP workload identity config json for trusted external cloud providers
+    creds: ""
```

Comment:
the name `creds` is somewhat ambitious

Reply:
Is there an expected naming suggestion? I used this to maintain the existing convention between TSA and Fulcio helm chart. How about `cloud_credential_config`?

Reply:
I am good with `cloud_credential_config`

Reply:
fixed!!